src/cozmo/world.py

```diff
@@ -136,7 +154,17 @@ def _allocate_object_from_msg(self, msg):
             logger.debug('Allocated object_id=%d to light cube %s', msg.objectID, cube)
             return cube
 
-        elif (msg.objectFamily == _clad_to_game_cozmo.ObjectFamily.CustomObject):
+        elif msg.objectFamily == _clad_to_game_cozmo.ObjectFamily.Charger:
+            charger = self.charger_factory(self.conn, self, msg.objectID, dispatch_parent=self)
+            if self.charger:
+                logger.error('Allocating multiple chargers: existing charger=%s msg=%s', self.charger, msg)
+            self.charger = charger
+            self._objects[charger.object_id] = charger
+            charger._robot = self.robot  # XXX this will move if/when we have multi-robot support
```

Comment:
I've always used TODO in the past, but seems that XXX is commonly used to point out odd but not necessarily broken (or to-be-done) things.